lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -5046,30 +5026,48 @@ int main(int argc, char *argv[]) {
     pause();
   }
 #endif
+
+  // Initialize LLDB first before we do anything.
+  lldb::SBDebugger::Initialize();
+
+  // Terminate the debugger before the C++ destructor chain kicks in.
+  auto terminate_debugger =
+      llvm::make_scope_exit([] { lldb::SBDebugger::Terminate(); });
+
+  DAP dap = DAP(program_path.str(), default_repl_mode);
```

Comment:
Should the dap object be copyable? Given that things hold references to it, maybe the copy operations should be deleted (and this object constructed in place) ?

Reply:
I can follow up with a delete of the copy constructor